src/query/storages/common/index/src/bloom_index.rs

```diff
@@ -556,7 +596,11 @@ fn visit_expr_column_eq_constant(
                 }
             }
             [
-                Expr::FunctionCall { id, args, .. },
+                Expr::FunctionCall { id, args, .. }
+                | Expr::Cast {
```

Comment:
This is incorrect, we can't support cast function in bf.

If a is int64
`where cast(a as int32) = 33i32` can't apply `a = 33i32`

Reply:
this is used to cast variant to string in equal compare, for example `where tags['uid'] = '123'`